qcodes/instrument_drivers/tektronix/Keithley_6500.py

```diff
@@ -149,6 +177,12 @@ def __init__(self, name, address, reset_device=False, **kwargs):
         self.write('FORM:DATA ASCII')
         self.connect_message()
 
+        command_set = self.ask_raw('*LANG?')
```

Comment:
i think this code should go above the `self.write` call. perhaps even to the top of the __init__ (right after the super __init__ call).

Also, no need for using ask_raw - just use ask :)

Reply:
✔️

Reply:
done